namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-staging/resources/sqs.tf

```diff
@@ -0,0 +1,14 @@
+module "laa_cla_backend_staging_sqs" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=4.0"
+
+  environment-name       = var.environment-name
+  team_name              = var.team_name
+  infrastructure-support = var.infrastructure-support
+  application            = var.application
+  sqs_name               = "cla_backend_staging_queue"
+  encrypt_sqs_kms        = "false"
+
+  providers = {
+    aws = aws.london
+  }
+}
```

Comment:
Along with the module, you need to create a kubernetes secret. 
Please have a look at the [example](https://github.com/ministryofjustice/cloud-platform-terraform-sqs/blob/master/example/sqs.tf).

Reply:
Thank you!

I've updated with suggested changes